Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
-name = "imap-proto"
-version = "0.1.0"
 authors = ["Dirkjan Ochtman <dirkjan@ochtman.nl>"]
+categories = ["authentication", "email", "encoding", "network-programming", "parser-implementations"]
 description = "IMAP protocol parser and data structures"
 documentation = "https://docs.rs/imap-proto"
-keywords = ["imap"]
-categories = ["email"]
 homepage = "https://github.com/djc/imap-proto"
+keywords = ["IMAP", "e-mail", "email"]
+license = "(Apache-2.0 OR MIT)"
```

Comment:
This is the first time I see this way of writing that, I think the way I had it before is more common (for example, Cargo itself has `MIT/Apache-2.0`).

Reply:
The old notation is ad hoc and its semantics are not standardized, the new is a true license expression. See this rust-lang/cargo#2171, esp. https://github.com/rust-lang/cargo/issues/2174#issuecomment-343526946. Can we use the new one?

Reply:
@djc this is the only remaining change TBD.

Reply:
I can see value in going with SPDX license expressions, but for now I'd prefer to stay with where the community largely is at the moment, and reconsider this at some point in the future.